src/playback/internal/au3/au3player.cpp

```diff
@@ -268,6 +268,11 @@ void Au3Player::stop()
     if (meter) {
         meter->Clear();
     }
+
+    while (!isBusy()) {
```

Comment:
The implementation of `isBusy` must be corrected, and then here have `while (isBusy())`, which is what makes sense.

Reply:
oops

Reply:
fixed